recipes/singularity-compose/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "singularity-compose" %}
+{% set version = "0.0.11" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 94eb17bab234d9c25afbee92970adc4e9400cf70dac9da6797b84e91974a09de
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - singularity-compose=scompose.client:start
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python >=3
+    - pytest-runner
+  run:
+    - python >=3
+    - requests
+    - pyaml
+    - spython
+
+test:
+  imports:
+    - scompose
+    - scompose.client
+    - scompose.client.build
+    - scompose.client.config
+    - scompose.client.create
+    - scompose.client.down
+    - scompose.client.exec
+    - scompose.client.logs
+    - scompose.client.ps
+    - scompose.client.restart
+    - scompose.client.shell
+    - scompose.client.up
+    - scompose.logger
+    - scompose.project
+    - scompose.project.project
+    - scompose.project.instance
+    - scompose.tests
+    - scompose.utils
+  commands:
+    - singularity-compose --help
+  requires:
+    - spython
```

Comment:
This will already be installed as it's a dependency of `singularity-compose`
```suggestion
```

Reply:
oup, just scrolled up :)